Makefile

```diff
@@ -0,0 +1,25 @@
+VERSION := patch
+GITHUB_API_TOKEN := ""
+
+release:
+	echo "Releasing version: $(VERSION)"
+	git checkout master
+	git pull origin master
+	npm run transpile
+	npm run dist
+	npm version $(VERSION)
+	npm publish
+	git push --follow-tags
+	npm run docs:publish
```

Comment:
we don't need all the commands, for example we don't use `gitbook`, so this line could be removed.